app/move/views/view.njk

```diff
@@ -219,45 +219,12 @@
 
       {% include "move/views/_includes/assessment.njk" %}
```

Comment:
Might it be cleaner to have:
```njk
{% include "move/views/_includes/assessment.njk" %}
{% include "move/views/_includes/court-information.njk" %}
{% include "move/views/_includes/documents.njk" %}
```
and move the conditionals to the chunks themselves?

Alternatively, could the conditionals be set up in the controller?

```njk
{% if includeCourtInformation %}
  {% include "move/views/_includes/court-information.njk" %}
{% endif %}
```
That might make it easier to unit test that we're outputting the right bits at least

Reply:
Yeah, ultimately I've trying to get to the latter that you've mentioned. I think it would be good to test those conditions via unit tests in the controller.

It has unfortunately evolved quite quickly with minimal large refactor so I was trying to pick and choose my battles. My next piece of work will involve changing how these sections are displayed quite drastically so you may see this evolve a bit more over the next couple of days...